ush/templates/FV3LAM_wflow.xml

```diff
@@ -369,11 +389,84 @@ MODULES_RUN_TASK_FP script.
     </dependency>
 
   </task>
+
+{%- if do_dacycle %}
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="&ANAL_GSI_INPUT_TN;" cycledefs="forecast" maxtries="2">
+
+    &RSRV_DEFAULT;
+    &WALL_LIMIT_ANAL;
+    <!--&WALL_LIMIT_WHOLE;-->
+
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&ANAL_GSI_INPUT_TN;" "&JOBSDIR;/JREGIONAL_RUN_ANAL"</command>
+
+    <nodes>16:ppn=24</nodes>
+    <walltime>00:30:00</walltime>
```

Comment:
Please use Jinja template here with values from the config settings.

Reply:
Done